### PR TITLE
fix: Add global focus style override

### LINKED ISSUE #1351: Header buttons don't have focus feedback
## Bug description
Header buttons don't have focus feedback which is important for accessibility purposes - https://www.outlinenone.com/

## Environment
 - Browser: Arc
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Go to https://app.safe.global/welcome
 2. Click tab and observe focus feedback on notifications center, connect button, network, etc

## Expected result
Buttons have focus feedback

## Obtained result
Buttons have no focus feedback

## Relevant links
https://www.outlinenone.com/

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves #1351 

## How this PR fixes it

- Adds a global style override for `focus-within` that works in webkit and firefox

## How to test it

1. Open the Safe
2. Press Tab multiple times
3. Observe all link/button elements have a focus style

## Screenshots
Screenshot 2022-12-23 at 11 18 44

### CODE REVIEW

**Comment by usame-algan** [comment]
Some elements are focused twice because they contain multiple nested links/buttons even though they don't need to. I created [another ticket](https://github.com/safe-global/web-core/issues/1450) to address this in order to keep this PR small

File: src/styles/globals.css [hunk 1]
```diff
@@ -36,6 +36,11 @@ button {
   font: inherit;
 }
 
+:focus-visible {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
```
Comment (usame-algan) [inline src/styles/globals.css:41]: Reference: https://css-tricks.com/copy-the-browsers-native-focus-styles/

**Review by katspaugh — APPROVED** [review]

**Comment by francovenica** [comment]
In this ticket I'll focus on the original report, which it was the headers. If there are more things to check I'll do it in the ticket mentioned by Usame in his comment

1 - The network button is not highlighted, what it is highlighted is the selection, which you can only see by clicking enter in the network selector button when the focus is on it. 
Will this be part of that 2nd ticket mentioned before?
tab selection

**Comment by katspaugh** [comment]
Fixed, @francovenica 👍

**Comment by francovenica** [comment]
Looks good!

**Review by mikhailxyz — APPROVED** [review]
Good job gentlemen 👍

**Pull request merged**